DESIGN.md

```diff
@@ -127,6 +139,24 @@ the operation):
 s3:buckets:1451635200000:foo-bucket:PutObject
 ```
 
+#### Local Redis Cache Key
+
```

Comment:
Can you expand the definition so that it's clear that this key is created in local cache and not Redis datastore.